server/item/item.go

```diff
@@ -107,6 +107,12 @@ type Weapon interface {
 	AttackDamage() float64
 }
 
+// CoolDown represents an item that has a cool down.
+type CoolDown interface {
```

Comment:
I would change the name of this interface, to maybe something like `CooledDown`, maybe not though because that doesn't make complete sense

Reply:
I don't know - would that not imply an item that currently has a cool down? To be fair, the current name is pretty awful, but I'm not sure what to call an item that can have a cool down.